### PR TITLE
ST-257 Allow merchant to process JSINIT

### PR DESCRIPTION
**Pull request by pgs-gmartowski**

### CODE REVIEW

**Comment by pgs-gmartowski** [comment]
@james-simpson-ST  Generally it can be review; I want to add this evening some small changes / refactor to classes which I has been changed.

**Comment by james-simpson-ST** [comment]
I agree with the implementation of this but I think the interface could be neater for the merchant so I think we should change it to be:
```
{ jwt: "CURRENTJWT",
  init: { "threedinit": "NEWCARDINALJWT",
           "cachetoken": "CACHETOKEN",
        }
} 
```

That way we keep all of these arguments for the init together in the config

**Comment by pgs-gmartowski** [comment]
> I agree with the implementation of this but I think the interface could be neater for the merchant so I think we should change it to be:
> 
> ```
> { jwt: "CURRENTJWT",
>   init: { "threedinit": "NEWCARDINALJWT",
>            "cachetoken": "CACHETOKEN",
>         }
> } 
> ```
> 
> That way we keep all of these arguments for the init together in the config

Good point !
I also noticed that I am only checking if cachetoken is specified but not threedinit ; changes in progress ...

File: example/bypass.html [hunk 1]
```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
```
[the remark this answers is not in the document]
Comment (james-simpson-ST) [reply example/bypass.html]: Yes I realized - great thinking and definitely something we needed :)

**Pull request merged**